data-pipeline/src/pipeline/pre_processing.py

```diff
@@ -666,32 +665,40 @@ def build_maintained_school_data(
 
     maintained_schools_list = pd.read_csv(
         maintained_schools_data_path,
-        encoding="utf8",
-        index_col=input_schemas.maintained_schools_master_list_index_col,
+        encoding="unicode-escape",
         usecols=input_schemas.maintained_schools_master_list.keys(),
-        dtype=input_schemas.maintained_schools_master_list,
     )
-
+    
+    mask = ((maintained_schools_list['Did Not Supply flag'] == '0').values 
```

Comment:
Could you run the linting commands over it?

```sh
poetry run black --config ./pyproject.toml src/ tests/
poetry run isort --settings-path ./pyproject.toml src/ tests/
```

Reply:
One day I will remember

Reply:
I need to add a check to the pipeline…